### ISSUE TITLE
Elements on page occlude navbar when scrolling - hiphop microworld in IE 11

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1001]
IE 11 on Win 7 VM, Production. The same issue doesn't seem to happen in Firefox 49 on Mac OS 10.11.

<img width="1039" alt="screen shot 2016-11-03 at 5 02 44 pm" src="https://cloud.githubusercontent.com/assets/1320456/20007621/716d88f4-a273-11e6-98cf-f188fb9c5e0f.png">
<img width="1075" alt="screen shot 2016-11-03 at 5 02 38 pm" src="https://cloud.githubusercontent.com/assets/1320456/20007620/716d7616-a273-11e6-8c36-a8271fa6b6ce.png">

**Comment by TheGrits** [comment]
Cannot reproduce using Internet Explorer 11 on a Windows 7 machine.

**Comment by chrisgarrity** [comment]
@mewtaylor Any ideas?

**Comment by mewtaylor** [comment]
@chrisgarrity yup – see https://github.com/LLK/scratchr2/issues/2725. Probably the same type of thing.
